lib/config/configtest/resolve_config_parser_test.go

```diff
@@ -0,0 +1,73 @@
+package configtest
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/chef/automate/lib/config"
+)
+
+func TestParseNameServers(t *testing.T) {
+	filecontent := []byte(`
+	nameserver 10.0.0.3
+	`)
+
+	ns := config.ParseNameServers(filecontent)
+	expectedResult := []string{"10.0.0.3"}
+	if !reflect.DeepEqual(expectedResult, ns) {
```

Comment:
We tend to use two small assertion libraries in our test that help with some of this:

https://godoc.org/github.com/stretchr/testify/assert
https://godoc.org/github.com/stretchr/testify/require

Reply:
Added "https://godoc.org/github.com/stretchr/testify/require" as the assertion library as suggested.